client/state/comments/ui/reducer.js

```diff
@@ -98,7 +98,7 @@ export const pendingActions = function ( state = [], action ) {
 		case COMMENTS_CHANGE_STATUS:
 		case COMMENTS_DELETE: {
 			const key = getRequestKey( action );
-			if ( has( action, 'meta.dataLayer.trackRequest' ) && state.indexOf( key ) === -1 ) {
+			if ( action.meta?.dataLayer?.trackRequest && ! state.includes( key ) ) {
```

Comment:
🤔  Won't that evaluate to `true` only if `trackRequest` is truthy? While previously, it was evaluating to `true` if the property existed, regardless of the value.

Reply:
Yes, and I think that's what we really want to check. When creating an action, the `meta.dataLayer.trackRequest` field is always set to `true`, or is not present at all. Nothing in between. And when the data layer code checks if it should track the request, it checks for truthiness:

https://github.com/Automattic/wp-calypso/blob/trunk/client/state/data-layer/wpcom-http/utils.js#L115

Reply:
That makes sense. Nice to see we're fixing bugs while doing those refactors 👍